autospec/build.py

```diff
@@ -196,20 +194,19 @@ def get_uniqueext(dirn, dist, name):
     return "{}-{}".format(name, seq)
 
 
-def set_mock():
-    global mock_cmd
-    # get group list of current user
-    user_grps = [grp.getgrgid(g).gr_name for g in os.getgroups()]
-    if os.path.exists('/usr/bin/mock'):
-        if 'mock' not in user_grps:
-            mock_cmd = 'sudo /usr/bin/mock'
+def get_mock_cmd():
+    if os.path.basename(os.path.realpath('/usr/bin/mock')) == 'mock':
+        return 'sudo /usr/bin/mock'
```

Comment:
On a misconfigured system, `util.call()` may get stuck awaiting input. So maybe `sudo` should be called with `-n` (`--non-interactive`) here to make sure it fails right away. Or is the intent is to provide an prompt in such scenario?

Reply:
Thanks for the observation (it will be useful for me in another context). I'm not sure what is the intent of autospec here, I've just kept the original behavior.